### PR TITLE
Adding login page

### LINKED ISSUE #94: Add login page
add simple login page.

### PR DESCRIPTION
**Pull request by idokk1**
Add Django built in login page. 
Because it's implemented by Django I didn't add tests for that (like login to admin page) except for test approaching /login url.
fix #94

### CODE REVIEW

File: home/tests.py [hunk 1]
```diff
@@ -423,3 +423,9 @@ def test_process_request_with_logged_out_user(self):
         request_mock = Mock(user=self.LoggedOutUserMock())
         pm.process_view(request_mock, None, None, None)
         assert request_mock.profile is None
+
+
+@pytest.mark.django_db
+def test_login(client):
+    response = client.get('/login')
+    assert response.status_code == 301
```
Comment (ifireball) [inline home/tests.py:431]: I'd expect a bit more then that - for example, where are you being redirected to?

**Review by ifireball — CHANGES_REQUESTED** [review]
Needs more tests, the fat that you are using Django's app is an implementation detail. How do you know login to your app actually works. The test should help you detect situations where you misconfigure Django or if a Django version update would break the functionality for you.

**Comment by idokk1** [comment]
Added another tests.

**Comment by avivz450** [comment]
LGTM

**Comment by danitLevi** [comment]
LGTM

**Review by ifireball — APPROVED** [review]

**Pull request merged**